docs/content/developers/dev-straat.md

```diff
@@ -1,104 +1,80 @@
-# Dev straat
+---
+title: "Ontwikkelstraat"
+description: ""
+weight: 30
+menu:
+  docs:
+    parent: "developers"
+---
 
-De developmentstraat is zoveel mogelijk geautomatiseerd zodat code-wijzigingen
-automatisch leiden tot het bijwerken van (test-)omgevingen.
+De ontwikkelstraat is zoveel mogelijk geautomatiseerd zodat code-wijzigingen automatisch leiden tot het bijwerken van (test-)omgevingen.
 
 Verder vinden er automatische checks plaats die de correctheid van
 implementaties en netheid van code bewaken.
 
-## Inhoud
-* [Github](#github)
-* [Jenkins](#jenkins)
-* [Docker Hub](#docker-hub)
-* [Kubernetes](#kubernetes)
-* [Swagger](#swagger)
-* [NLX Directory](#nlx-directory)
-* [Docker-compose](#docker-compose)
 
 **Architectuurschets**
 
-![DevOps pipeline](./bestanden/dev-straat.png)
+![DevOps pipeline](../bestanden/dev-straat.png)
 
 ## Github
 
 Alle ontwikkeling vindt plaats op Github. Het centrale punt is de
 [gemma-zaken](https://github.com.com/VNG-Realisatie/gemma-zaken) repository.
 
-Het one-flow git-branching model wordt hier aangehouden - dit betekent dat de
-`master` branch altijd de 'huidige waarheid' is, en aanpassingen worden via
-feature branches geïntroduceerd.
+Het one-flow git-branching model wordt hier aangehouden - dit betekent dat de `master` branch altijd de 'huidige waarheid' is, en aanpassingen worden via feature branches geïntroduceerd.
 
-Daarnaast zijn er repositories voor de referentie-implementaties en
-ondersteunende tooling.
+Daarnaast zijn er repositories voor de referentie-implementaties en ondersteunende tooling.
 
 ### Referentie-implementaties
 
-Iedere referentieimplementatie van een component leeft in zijn eigen
-repository:
+Iedere referentieimplementatie van een component leeft in zijn eigen repository:
 
 * [ZRC](https://github.com/vng-Realisatie/gemma-zaakregistratiecomponent)
 * [DRC](https://github.com/VNG-Realisatie/gemma-documentregistratiecomponent)
 * [ZTC](https://github.com/VNG-Realisatie/gemma-zaaktypecatalogus)
 
-Voor de referentieimplementaties wordt het git-flow git-branching model
-aangehouden. Dit houdt in dat de `master` branch de huidige stabiele,
-'productie' versie is en de `develop` branch de volgende versie wordt.
-Wijzigingen komen via feature branches in `develop` terecht. Releases worden
-getagged om ze te merken als een stable release, waarbij semantic versioning
-gebruikt wordt.
+Voor de referentieimplementaties wordt het git-flow git-branching model aangehouden. Dit houdt in dat de `master` branch de huidige stabiele, 'productie' versie is en de `develop` branch de volgende versie wordt.
```

Comment:
met het risico dat ik nu heel erg ga zeuren - maar het samenvoegen van deze zinnen op 1 lijn maken het best vervelend in code-editors op gebied van line-wrapping. Typisch wordt een lijnlengte van 80-120 karakters aangehouden.

Reply:
Ah, prima, zal het wijzigen. Gebruik zelf dit: http://blog.atom.io/2014/10/31/language-scoped-config.html